README.md

```diff
@@ -34,7 +34,29 @@ php artisan socialstream:install
 The `socialstream:install` command will overwrite the Jetstream published files which are required for Socialstream to work. 
 
 > Note: If you don't have Laravel Jetstream installed, the above command will walk you through the steps required to install it.
-
+> 
+## Configuration & Setup
+Once Socialstream is installed, it will publish a config file. In this config file, you can define whether or not the packages alterations should be shown, the middleware used to wrap the routes as well as the providers that you wish to use:
+```
+<?php
+return [
+  'show' => true,
+  'middleware => ['web'],
+  'providers => [
+    'github',
+    'facebook',
+    'google'
+  ],
+];
+```
+Once you’ve defined your providers, you will need to provider client_id , client_secret and redirect keys for each in your services.php config file. E.g.
```

Comment:
Can the wording of this be changed to

“once you’ve defined your providers, you will need to update your `services.php` to create `client_id`, `client_secret` and `redirect` keys for each provider:”